src/components/portrait/portrait.pw.tsx

```diff
@@ -0,0 +1,486 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import { getDataElementByValue } from "../../../playwright/components";
+import {
+  portraitImage,
+  portraitInitials,
+  portraitPreview,
+} from "../../../playwright/components/portrait/index";
+
+import { icon } from "../../../playwright/components/index";
+
+import {
+  CHARACTERS,
+  COLOR,
+  SIZE,
+  VALIDATION,
+} from "../../../playwright/support/constants";
+import { checkAccessibility } from "../../../playwright/support/helper";
+import Box from "../../../src/components/box";
+import {
+  PORTRAIT_SIZE_PARAMS,
+  PORTRAIT_SIZES,
+} from "../../../src/components/portrait/portrait.config";
+import {
+  PortraitComponent,
+  PortraitDefaultComponent,
+} from "./components.test-pw";
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+const colors = [
+  ["orange", COLOR.ORANGE],
+  ["red", COLOR.RED],
+  ["black", COLOR.BLACK],
+  ["brown", COLOR.BROWN],
+];
+const testImage =
+  "https://www.gravatar.com/avatar/cceff1ad774bf89b1f75cb6e5005333c?s=40&d=404";
+const imageURL = [
+  "https://avataaars.io/?avatarStyle=Transparent&topType=LongHairStraight&accessoriesType=Blank&hairColor=BrownDark&facialHairType=Blank&clotheType=BlazerShirt&eyeType=Default&eyebrowType=Default&mouthType=Default&skinColor=Light",
+  "https://www.gravatar.com/avatar/05c1c705ee45d7ae88b80b3a8866ddaa?s=24&d=404",
+];
+
+const portraitSizes = PORTRAIT_SIZES.map((size) => [
+  size,
+  PORTRAIT_SIZE_PARAMS[size].dimensions,
+]);
+
+const visibilityValues: [boolean, string][] = [
+  [true, "with"],
+  [false, "without"],
+];
+
+test.describe("Portrait component", () => {
+  test("should render with gravatar prop", async ({ mount, page }) => {
+    await mount(<PortraitDefaultComponent gravatar="chris.barber@sage.com" />);
+
+    await expect(portraitPreview(page)).toHaveCSS("height", "40px");
+    await expect(portraitPreview(page)).toHaveCSS("width", "40px");
+    await expect(portraitPreview(page).locator("img")).toHaveAttribute(
+      "src",
+      testImage
+    );
+  });
+
+  imageURL.forEach((urlVals) => {
```

Comment:
```suggestion
  imageURL.forEach((url) => {
```

**suggestion**: each element of `imageURL` is just a url (or a string represetting one), so why not just call it that? The original Cypress test used `scrImage`, which I think is fine - and I don't even mind `urlVal` if you really insist. What I definitely don't want is to use a plural (as you have with `urlVals`), when each one is a single URL/string/image.

On a similar note I dislike `imageURL` for the array because there's more than one, I'd call it `imageURLs` myself (or similar). But I realise that in the Cypress test it also had this name, so while I would change it personally I don't feel as strongly about it.